src/components/HURUmap/LocationHeader/index.js

```diff
@@ -0,0 +1,47 @@
+import { Typography, Grid, Button } from "@material-ui/core";
+import Icon from "next/image";
+import PropTypes from "prop-types";
+import React from "react";
+
+import useStyles from "./useStyles";
+
+const LocationHeader = ({ level, parent, title, icon, ...props }) => {
+  const classes = useStyles(props);
+
+  if (!(title && parent)) {
+    return null;
+  }
+  return (
+    <div className={classes.root}>
+      <Grid container justifyContent="space-between">
+        <Typography variant="h3" className={classes.title}>
```

Comment:
1. If you prefer using `Grid`, then surround it's children with `<Grid item>` i.e. `<Grid container ...><Grid item><Typography ...`
2. Otherwise, plain `div` with styling in css or `<Box display="flex" justifyContent="space-between"><Typography ...` will do